testing/functional/tests/cypress/page-objects/student-objects/ApplicationObject.ts

```diff
@@ -0,0 +1,279 @@
+      cy.get(".form-group").eq(4).click({ force: true });
+      cy.wait(1000);
+      cy.get(".choices__input.choices__input--cloned")
+        .eq(1)
+        .click({ force: true })
+        .type(testData.howWillYouAttendProgram2)
+        .type("{enter}");
+    });
+    cy.wait(2000);
+  }
+
+  programIWillBeAttendingDropdown() {
+    cy.fixture("draftApplicationData").then((testData) => {
+      cy.get(".form-group").eq(6).click({ force: true });
+      cy.wait(1000);
+      cy.get(".choices__input.choices__input--cloned")
+        .eq(2)
+        .click({ force: true })
+        .type(testData.programIWillBeAttendingDropdown)
+        .type("{enter}");
+    });
+    cy.wait(2000);
+  }
+
+  programIWillBeAttendingDropdown2() {
+    cy.fixture("draftApplicationData").then((testData) => {
+      cy.get(".form-group").eq(6).click({ force: true });
+      cy.wait(1000);
+      cy.get(".choices__input.choices__input--cloned")
+        .eq(2)
+        .click({ force: true })
+        .type(testData.programIWillBeAttendingDropdown2)
+        .type("{enter}");
+    });
+    cy.wait(2000);
+  }
+
+  myStudyPeriodIsNotListedCheckbox() {
+    return cy.get("#eoslne6-offeringnotListed");
+  }
+
+  programName(){
+    return cy.get("#ek2u9li-programName");
+  }
+
+  programDescription(){
+    return cy.get("e1w8o5a-programDescription")
+  }
+
+  studyStartDate() {
+    cy.fixture("draftApplicationData").then((testData) => {
+      cy.get('#eqnahab > [ref="element"] > .input-group > .input').type(
+        testData.studyStartDate
+      );
+    });
+    cy.wait(2000);
+  }
+
+  studyEndDate() {
+    cy.fixture("draftApplicationData").then((testData) => {
+      cy.get('#eyw0exi > [ref="element"] > .input-group > .input').type(
+        testData.studyEndDate
+      );
+    });
+    cy.wait(2000);
+  }
+
+  inputStudentNumber() {
+    cy.fixture("draftApplicationData").then((testData) => {
+      cy.get("#ek8w4hn-studentNumber").type(testData.studentNumber);
+    });
+    cy.wait(2000);
+  }
+
+  inputStudentNumber2() {
+    cy.fixture("draftApplicationData").then((testData) => {
+      cy.get("#ek8w4hn-studentNumber").type(testData.studentNumber2);
+    });
+    cy.wait(2000);
+  }
+
+  incorrectStudentNumber() {
+    return cy.get("#ek8w4hn-studentNumber");
+  }
+
+  incorrectStudentNumberText() {
+    return cy.contains("Student number must have no more than 12 characters.");
+  }
+
+  programOfferingDropdown() {
+    cy.fixture("draftApplicationData").then((testData) => {
+      cy.xpath(
+        "//div[@id='eb4inb']//div[@class='form-control ui fluid selection dropdown']"
+      ).click({ force: true });
+      cy.wait(1000);
+      cy.get(".choices__input.choices__input--cloned")
+        .eq(3)
+        .click({ force: true })
+        .type(testData.programOffering)
+        .type("{enter}");
+    });
+    cy.wait(2000);
+  }
+  //<EndRegion--------------------------Draft Status------------------------------------->
+
+  ProgramPageText() {
```

Comment:
Please check the methods cases to be camel cases.

Reply:
Changes have been done.